guides/source/form_helpers.md

```diff
@@ -121,24 +135,26 @@ Radio buttons, while similar to checkboxes, are controls that specify a set of o
 <%= form.label :age_adult, "I am over 21" %>
 ```
 
-Output:
+Will generate the following HTML:
```

Comment:
Yeah I noticed this...it is capital in tens of other code snippets so I followed suit for consistency.

Reply:
I think capital in this case looks better (and like Bhumi said, is prevalent throughout the guides) but I would try to avoid partial sentences as much as possible and use full sentences.

**For instance**:
The above will generate the following HTML: